.github/workflows/backend-ci.yaml

```diff
@@ -0,0 +1,90 @@
+name: Backend build
+
+on:
+    push:
+        branches:
+            - main
+            - '[0-9]+.[0-9]+'
+    pull_request: ~
+
+jobs:
+    cs-fix:
+        name: Run code style check
+        runs-on: "ubuntu-22.04"
+        strategy:
+            matrix:
+                php:
+                    - '8.1'
```

Comment:
Shouldn't we run CS check on 8.3? Or is it due to older code-style requirement?

Reply:
It's in-tact with how we do things for 4.6, just tried to align.